app/lib/common/widgets/md_editor_with_preview.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:acter/common/providers/sdk_provider.dart';
+import 'package:acter/common/widgets/render_html.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+final markdownProvider =
```

Comment:
Do we want to keep it publicly accessible? Otherwise have it labelled as ```private```. The convention is good to avoid conflicting names which could reduce possibility of introducing bugs.

Reply:
I think it can be publicly accessible, that is fine for anything that wants to render markdown->html in the backend manner.